20150310-wp-core-contrib-day.md

```diff
@@ -0,0 +1,255 @@
+
+### Needs Unit Tests
+[https://core.trac.wordpress.org/tickets/needs-unit-tests](https://core.trac.wordpress.org/tickets/needs-unit-tests)
+
+Are you handy with Unit Tests? Maybe you're looking to explore WordPress Core and contribute at the same time? Then the "Needs Unit Tests" report is just the ticket!
+
+### Next Major and Next Minor Release
+[https://core.trac.wordpress.org/tickets/major/workflow](https://core.trac.wordpress.org/tickets/major/workflow)
+[https://core.trac.wordpress.org/tickets/minor/workflow](https://core.trac.wordpress.org/tickets/minor/workflow)
+
+These catch-all reports are a great way to target your efforts on tickets assigned to the next couple of releases. The above links are versions of the [Next Major](https://core.trac.wordpress.org/tickets/major) and and [Next Minor](https://core.trac.wordpress.org/tickets/minor) Release reports grouped by Workflow Status, I prefer these as it is much easier to find tickets that match your skills (development, testing etc).
+
+### Bug Reports
+[https://make.wordpress.org/core/reports/](https://make.wordpress.org/core/reports/)
+
+This report has a mis-leading title, it's more of a **Tickets Dashboard** and gives you a great jumping off point to various ticket reports, it's the place to go to when you want to browse a little.
+
+There's a nice "Tickets by Topic" section that gives you quick access to the Components and Focus Areas for WordPress, for us a natural pick is the "[Plugins](https://core.trac.wordpress.org/component/Plugins)" Component area.
+
+There's also a "Me" section that is very useful for tracking tickets you've created, commented on or favourited. You can also see any tickets that you have outstanding patches on.
+
+## Setting Up a Development Site
+OK, you've found a ticket to investigate, to get any further you need to download the WordPress Core source code, build WordPress, configure your local development site and give it a spin before diving in to development.
+
+### Download The Source
+The WordPress Core source is very different to the WordPress software you download to set up a new instance of WordPress, it has a different directory structure to help with development. WordPress Core is held in a Subversion repository, as such you should use Subversion to obtain your copy of trunk.
+
+```
+svn co https://develop.svn.wordpress.org/trunk wordpress-core
+```
+
+You should see a directory structure similar to the following:
+
+![WordPress Core Directory Structure](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/WordPress-Core-Directory-Structure.png)
+
+### Build WordPress
+You have now checked out the WordPress source, but as it stands you can not spin up a WordPress site just yet, first you need to build WordPress.
```

Comment:
Are you sure? I just checked out latest trunk and was able to run it without doing a build.